app/models/lead_provider.rb

```diff
@@ -25,6 +25,7 @@ class LeadProvider < ApplicationRecord
   has_many :partnership_csv_uploads
   has_many :lead_provider_api_tokens
   has_one :call_off_contract
```

Comment:
as discussed this seems wrong and it is, when we call this it just picks one contract for lead provider, checking the code it is called BUT not used anywhere except in banding tracker which is deprecated and not built correctly/used. We can add a tech debt ticket to remove this